sbom/sbom.go

```diff
@@ -124,6 +124,26 @@ func GenerateBom(r *ReportCollectionJson) ([]Sbom, error) {
 						})
 					}
 
+					bom.Packages = append(bom.Packages, bomPkg)
+				}
+			}
+			if rb.NpmPackages != nil {
```

Comment:
you don't need to check for nil before iterating:

>For an array, pointer to array, or slice value a, the index iteration values are produced in increasing order, starting at element index 0. If at most one iteration variable is present, the range loop produces iteration values from 0 up to len(a)-1 and does not index into the array or slice itself. For a nil slice, the number of iterations is 0.

Reply:
Great point, useless if conditions removed.